addon/src/lib/reducers/sideEffects.js

```diff
@@ -78,11 +76,14 @@ export function reducer(
     case actions.EXPERIMENTS_LOADED.type:
       return ({ dispatch, loader }) => {
         loader.schedule();
-        Object.keys(payload.experiments)
+        Object
+          .keys(payload.experiments)
           .map(id => payload.experiments[id])
           .filter(x => x.uninstalled && new Date(x.uninstalled) < new Date())
-          .forEach(experiment => dispatch(actions.UNINSTALL_EXPERIMENT({ experiment })))
-      }
+          .forEach(
+            experiment => dispatch(actions.UNINSTALL_EXPERIMENT({ experiment }))
```

Comment:
Is this like an 80 chars max thing? If we're going to start enforcing line lengths, I think it'd be good to work out a standard as a team and add something to the eslintrc.